README.md

```diff
@@ -5,172 +5,46 @@
-
-```php
-# Both call will send the "app:php1" and "beta" tags.
-$statsd->increment('your.data.point', 1, array('app' => 'php1', 'beta' => null));
-$statsd->increment('your.data.point', 1, "app:php1,beta");
-```
-
-### Increment
-
-To increment things:
-
-``` php
-$statsd->increment('your.data.point');
-$statsd->increment('your.data.point', .5);
-$statsd->increment('your.data.point', 1, array('tagname' => 'value'));
-```
-
-### Decrement
-
-To decrement things:
-
-``` php
-$statsd->decrement('your.data.point');
-```
-
-### Timing
-
-To time things:
-
-``` php
-$start_time = microtime(true);
-run_function();
-$statsd->microtiming('your.data.point', microtime(true) - $start_time);
-
-$statsd->microtiming('your.data.point', microtime(true) - $start_time, 1, array('tagname' => 'value'));
-```
-
-### Submitting events
 
-For documentation on the values of events, see
-[http://docs.datadoghq.com/api/#events](http://docs.datadoghq.com/api/#events).
-
-#### Submitting events via TCP vs UDP
-
-- **TCP** - High-confidence event submission. Will log errors on event submission error.
-- **UDP** - "Fire and forget" event submission. Will **not** log errors on event submission error. No acknowledgement of submitted event from Datadog.
-
-No matter wich transport is used the `event` function has the same API.
-
-_[Differences between TCP/UDP](http://stackoverflow.com/a/5970545)_
-
-##### UDP Submission to local dogstatsd
-
-Since the UDP method uses the a local dogstatsd instance we don't need to setup any additional application/api access.
-
-```php
-$statsd = new DogStatsd();
-$statsd->event('Fire and forget!',
-    array(
-        'text'       => 'Sending errors via UDP is faster but less reliable!',
-        'alert_type' => 'success'
-    )
-);
-```
-
-- Default method
-- No configuration
-- Faster
-- Less reliable
-- No logging on communication errors with Datadog (fire and forget)
-
-##### TCP Submission to Datadog API
-
-To submit events via TCP, you'll need to first configure the library with your
-Datadog credentials, since the event function submits directly to Datadog
-instead of sending to a local dogstatsd instance.
-
-You can find your api and app keys in the [API tab](https://app.datadoghq.com/account/settings#api).
-
-```php
 $statsd = new DogStatsd(
-    array('api_key' => 'myApiKey',
-          'app_key' => 'myAppKey',
+    array('host' => '127.0.0.1',
+          'port' => 8125,
      )
   );
-
-$statsd->event('A thing broke!',
-    array(
-        'alert_type'      => 'error',
-        'aggregation_key' => 'test_aggr'
-    )
-);
-$statsd->event('Now it is fixed.',
-    array(
-        'alert_type'      => 'success',
-        'aggregation_key' => 'test_aggr'
-    )
-);
 ```
 
-- Slower
-- More reliable
-- Logging on communication errors with Datadog (uses cURL for API request)
-- Logs via error_log and try/catch block to not throw warnings/errors on communication issues with API
```

Comment:
This is a specificity of the PHP client and it not describe in the doc website (don't ask me why we handle events like that ...). We should keep it too.

Reply:
🤔 do you know if it's used often ? I'm going to create a FAQ just for this as it's specific with PHP

Reply:
https://github.com/DataDog/documentation/pull/5954

Reply:
We have no way to know how use it is, but we had a few customers contact us on the support that where using this. So it's "used".